app/controllers/apidocs_controller.rb

```diff
@@ -57,4 +57,6 @@ class ApidocsController < ApplicationController
   def index
     render json: Swagger::Blocks.build_root_json(SWAGGERED_CLASSES)
   end
+
+  def swagger_ui; end
```

Comment:
- create a separate controller  (swagger_ui_controller) with index action
- need to test the route & controller too